cmd/docker-app/status.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/deislabs/duffle/pkg/action"
+	"github.com/deislabs/duffle/pkg/claim"
+	"github.com/deislabs/duffle/pkg/credentials"
+	"github.com/deislabs/duffle/pkg/utils/crud"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func statusCmd(dockerCli command.Cli) *cobra.Command {
+	var opts credentialOptions
+
+	cmd := &cobra.Command{
+		Use:   "status <installation-name>",
+		Short: "Get the installation status. If the installation is a docker application, the status shows the stack services.",
+		Args:  cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runStatus(dockerCli, args[0], opts)
+		},
+	}
+	opts.addFlags(cmd.Flags())
```

Comment:
`runStatus` uses `opts.targetContext`, so do we not need to call `opts.creds.addflags` here (same for some other subcommands it seems)?

Reply:
Well opts is a `credentialOptions`, so calling `addFlags` on it adds the `credential-set` flags already.

Reply:
My mistake, thought this was the same as the `install` case with a containing opts.